static/js/subs.js

```diff
@@ -3,10 +3,16 @@ var subs = (function () {
 var exports = {};
 
 function add_admin_options(sub) {
+    if (sub.subscribed == true) {
+        var can_moderate = _.contains(sub.permissions, 'can_moderate');
+        var can_moderate_stream = page_params.is_admin || can_moderate;
+        var can_add_remove_users = _.contains(sub.permissions, 'can_add_remove_users');
+    }
     return _.extend(sub, {
```

Comment:
The codepath below is hit even when `sub.subcribed` is false, in which case `can_moderate_stream` is undefined